nemo/collections/asr/parts/mixins/mixins.py

```diff
@@ -214,7 +214,12 @@ def _setup_aggregate_tokenizer(self, tokenizer_cfg: DictConfig):
                         self.AGGREGATE_TOKENIZERS_DICT_PREFIX
                     ][lang]['type']
 
-        self.tokenizer = tokenizers.AggregateTokenizer(tokenizers_dict)
+        if tokenizer_cfg.get('is_canary', False):
```

Comment:
If `custom_tokenizer` in tokenizer_cfg:
  tokenizer = self.from_config_dict(tokenizer_cfg.custom_tokenizer)
Else:
  Tokenizer = AGGTokenzier()

Infact I think this can be populated even above so it becomes general to any type of tokenizer but for now keep it here I will refactor later 

Inside the config for tooenizers it will be like this 

```yaml
...
tokenizer:
  dir: xyz
  type: xyz
  custom_tokenizer:
    __target__: ABC.XYZ.CanaryTokenizer
    Kwarg1: UVW
...
```

Reply:
@titu1994  TODO